global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.101",
+    "version": "8.0.100",
```

Comment:
will revert this - just updated the visual studio and the dotnet sdk 7 was removed.